test/quantization/bc/test_backward_compatibility.py

```diff
@@ -42,8 +48,25 @@ def get_filenames(self, subname):
     scripted_module_file = base_name + ".scripted.pt"
     traced_module_file = base_name + ".traced.pt"
     expected_file = base_name + ".expected.pt"
+    package_file = base_name + ".package.pt"
+    get_attr_targets_file = base_name + ".get_attr_targets.pt"
+
+    return input_file, state_dict_file, scripted_module_file, \
+        traced_module_file, expected_file, package_file, get_attr_targets_file
+
+
+class LinearReluFunctional(nn.Module):
```

Comment:
For the model it might be useful to introduce some kind of a hierarchy, to make sure the attr FQNs correctly reflect that. Similar to https://github.com/pytorch/pytorch/blob/master/test/quantization/fx/test_quantize_fx.py#L2151

Reply:
sure